src/AnonymizationStep/AnonymizedResultsTable.tsx

```diff
@@ -114,17 +120,17 @@ const mapColumn =
     if (column.type === 'aggregate') {
       switch (mode) {
         case 'anonymized':
-          return [makeColumnData(column.name, columnIdx + '_anon', AGG_COLUMN_TYPE, renderValue)];
+          return [makeColumnData(column.name, `${columnIdx}_anon`, AGG_COLUMN_TYPE, renderValue)];
         case 'combined':
           return [
-            makeColumnData(column.name + ' (anonymized)', columnIdx + '_anon', AGG_COLUMN_TYPE, renderLowCountValue),
-            makeColumnData(column.name + ' (original)', columnIdx + '_real', AGG_COLUMN_TYPE, renderValue),
-            makeColumnData('Distortion', columnIdx + '_diff', 'real', renderRelativeNoiseValue),
+            makeColumnData(column.name + ' (anonymized)', `${columnIdx}_anon`, AGG_COLUMN_TYPE, renderLowCountValue),
+            makeColumnData(column.name + ' (original)', `${columnIdx}_real`, AGG_COLUMN_TYPE, renderValue),
+            makeColumnData('Distortion', `${columnIdx}_diff`, AGG_COLUMN_TYPE, renderRelativeNoiseValue),
```

Comment:
Distortion is always real. But why is count a real I don't know... Either way doesn't affect anything.

Reply:
dunno, I just made this consistent :shrug:

Reply:
I'd keep this `real` because it's not an aggregated column, it's statistical information. As for aggs I suppose real is a superset of integer (in JS land there's no int).